Blob.js

```diff
@@ -352,10 +352,10 @@
 						chunks[i] = chunk._buffer;
 					} else if (typeof chunk === "string") {
 						chunks[i] = textEncode(chunk);
-					} else if (arrayBufferSupported && (Object.prototype.isPrototypeOf.call(ArrayBuffer, chunk) || isArrayBufferView(chunk))) {
+					} else if (arrayBufferSupported && (Object.prototype.isPrototypeOf(ArrayBuffer, chunk) || isArrayBufferView(chunk))) {
```

Comment:
Likewise, this now tests if the ArrayBuffer class is an Object and is always true

It can be fixed in the same way:
```suggestion
					} else if (arrayBufferSupported && (Object.prototype.isPrototypeOf(ArrayBuffer.prototype, chunk) || isArrayBufferView(chunk))) {
```

Reply:
Thanks, change made